examples/extended/grails-app/services/test/TestDataService.groovy

```diff
@@ -158,7 +165,7 @@ class TestDataService {
 	}
 
 	protected save(instance) {
-		instance.save(failOnError: true)
+		instance.save(failOnError: true, flush: true)
```

Comment:
Why does this need `flush: true`?  It's best to avoid flushing if at all possible.

Reply:
fixed in next commit